src/app/containers/ArticleMain/index.jsx

```diff
@@ -8,23 +8,46 @@ import Blocks from '../Blocks';
 import timestamp from '../ArticleTimestamp';
 import { GhostWrapper } from '../../lib/styledGrid';
 import ATIAnalytics from '../ATIAnalytics';
+import audioVideo from '../AudioVideo';
+import AudioVideoHead from '../../components/AudioVideoHead';
+import { RequestContext } from '../../contexts/RequestContext';
+import generateAVSettings from '../../lib/utilities/audioVideo/generateAVSettings';
 
 const componentsToRender = {
   headline: headings,
   subheadline: headings,
   text,
   image,
   timestamp,
+  audio: audioVideo,
+  video: audioVideo,
 };
 
 const ArticleMain = ({ articleData }) => {
+  const {
+    env,
+    platform,
+    statsDestination,
+    statsPageIdentifier,
+  } = React.useContext(RequestContext);
   const { content, metadata, promo } = articleData;
   const { blocks } = content.model;
 
   return (
     <Fragment>
       <ATIAnalytics data={articleData} />
       <MetadataContainer metadata={metadata} promo={promo} />
+      {platform === 'canonical' ? (
+        <AudioVideoHead
+          audioVideoAssets={generateAVSettings({
+            blocks,
+            env,
+            platform,
+            statsDestination,
+            statsPageIdentifier,
+          })}
+        />
+      ) : null}
```

Comment:
Are we not doing AMP in this issue?

Reply:
We are not including AMP video/audio in this issue - it will be added in #1764.